test cases/rust/1 basic/meson.build

```diff
@@ -17,4 +17,5 @@ test(
     override_options : ['b_ndebug=false'],
   ),
   should_fail : true,
+  protocol : default(),
```

Comment:
I would like to see a unit test that verify passing `default()` to a new kwarg does not trigger FeatureNew/FeatureDeprecated warnings.